src/Api.js

```diff
@@ -80,17 +76,28 @@ export default class Api extends ToolApi {
   * @param {Boolean} warnOnError - if true, then will show message on selection change
   * @return {Function}
   */
-  validateVerse(targetVerse, skipCurrent = false, contextId = null, warnOnError = false) {
+  validateVerse(targetVerse, skipCurrent = false, contextId = null, ) {
```

Comment:
This method must have the following signature in order to be compatible with tc triggering verse validation:

* `validateVerse(chapter, verse)` where `chapter` and `verse` is a number.

This is how tC can trigger all tools to perform a validation. e.g. when a verse is edited tC will execute `api.trigger('validateVerse', chapterWithVerseEdit, verseWithVerseEdit)`